GETTING_STARTED.md

```diff
@@ -0,0 +1,240 @@
+# RancherOS Getting Started Guide
+
+RancherOS is a lightweight Linux distro designed from the ground up to run Docker containers. This guide will provide detailed instructions on how to get started using RancherOS, and will include:
+  - An overview of the RancherOS architecture.
+  - Installation instructions for running RancherOS on Vagrant or AWS. 
+  - Overview of how to deploy Docker containers on RancherOS.
+  - Instructions for deploying system containers.
+  - Details on how to use RancherOS with Rancher management software.
+
+## RancherOS Architecture
+
+RancherOS is a 20MB distro which runs the latest Docker daemon as PID1, the first process on the system.  All other system services, like ntpd, rsyslog, console, etc. are run as Docker containers.
+
+The first process on the system is called **System Docker** and its responsible for managing the system services on RancherOS, this process replaces traditional init systems like systemd, and can be used to launch additional system services, which we will see later in this guide..
```

Comment:
its responsible -> it's responsible.

RancherOS. (full stop) System Docker replaces ...